public/favicon/site.webmanifest

```diff
@@ -3,14 +3,14 @@
   "short_name": "",
   "icons": [
     {
-      "src": "/android-chrome-192x192.png",
+      "src": "/android-chrome-192x192.webp",
```

Comment:
**issue:** @TenzDelek, quick question regarding icons. In public/favicon I see several sets of icons:
- android-chrome*
- apple-touch*
- favicon*

From what I can see, favicon* are never used in code as well as DearDiary image. Should I still keep them or unused images should be deleted?